docs/concepts/copilot/enable-copilot.md

```diff
@@ -12,21 +12,22 @@ tags:
 
 From the kubernetes cluster where Prophecy services are running:
 
-1. The `prophecy-operator` service must be running version `3.1.0.0` or greater.
-2. The `athena` Prophecy service should be running version `3.1.0.0` or greater. This is only required if using the Pipeline approach to the upgrade.
-3. In the `ProphecyCluster` CR, do the following:
+1. The `prophecy-operator`, `edweb`, `metagraph`, and other Prophecy services must be running version `3.1.0.0` or greater.
+2. The above services should all be running the same release-version as the `copilot` service defined in the image below, eg `3.1.0.0`
+3. The `athena` Prophecy service should be running version `3.1.0.0` or greater. This is only required if using the Pipeline approach to the upgrade.
+4. In the `ProphecyCluster` CR, do the following:
 
 - add `copilot` to `enabledapps`
 - add `image` section if required as below, else defaults to `latest`
 - add the `copilot` section as below in the same level as other enabled apps
 
 ```
 copilot:
-    image: 133450206866.dkr.ecr.us-west-1.amazonaws.com/copilot:3.1.0.0
+    image:  gcr.io/prophecy-share/copilot:<release-version>
```

Comment:
I don't know how to make the "p" lowercase if I change this to "Prophecy-release-version"
@hareshkh @prashanth26  if that's ok with you, pls approve this PR.